frontend/src/Home.js

```diff
@@ -45,7 +45,7 @@ export const Home = () => (
         <FilesToUpload path="/p1/uploadfilesquestion" />
         <FileUploadPage path="/p1/uploadfiles" />
         <Willbecontacted path="/p1/contactinfoquestion" />
-        <ContactInfoPage path="/p1/contactinfo" />
+        <P2ContactInfoPage path="/p2/contactinfo" />
```

Comment:
ah, we need to leave in the p1 page as well :)

Reply:
This makes sense